src/transformState.js

```diff
@@ -5,7 +5,21 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
you may use clear naming here